content/applications/marketing/events/event_templates.rst

```diff
@@ -0,0 +1,217 @@
+===============
+Event templates
+===============
+
+The Odoo *Events* application provides users with the ability to customize and configure event
+templates, which can be used to expedite the event-creation process.
+
+These templates can be created and personalized in the application, and then selected from an event
+form, in order to quickly apply a series of settings and elements to the new event, all of which can
+be further modified, if needed.
+
+Event templates page
+====================
+
+In the Odoo *Events* application, event templates can quickly be created and modified.
+
+To begin, navigate to :menuselection:`Events app --> Configuration --> Event Templates`. Doing so
+reveals the :guilabel:`Event Templates` page. Here, users can find all the existing event templates
+in the database.
+
+.. image:: event_templates/event-templates-page.png
+   :align: center
+   :alt: The Event Templates page in the Odoo Events application.
+
+By default, Odoo provides users with three pre-configured event templates: :guilabel:`Exhibition`,
```

Comment:
Same here: don't need "users" or any other pronouns to complete the sentence.
Consider removing doc-wide.

```suggestion
By default, Odoo provides three pre-configured event templates: :guilabel:`Exhibition`,
```